pkg/kvstore/allocator/allocator.go

```diff
@@ -723,7 +729,7 @@ func (a *Allocator) RunGC() error {
 			if err := kvstore.Delete(key); err != nil {
 				scopedLog.WithError(err).Warning("Unable to delete unused allocator master key")
 			} else {
-				scopedLog.Info("Deleted unused allocator master key")
+				scopedLog.Info("Released unused identity")
```

Comment:
What is the difference here semantically between "allocator master key" vs. "identity"?

Reply:
Nothing really, in fact, the original message was better. Fixed